Utils/constants.js

```diff
@@ -3,6 +3,15 @@ const TalentBoardUserStatus = {
   EXPIRED: 2
 };
 
+const VisibilityDurationMapping = {
```

Comment:
can you also add a comment explaining about this constant

Reply:
done